http/src/main/resources/init-resources/azure_vm_init_script.sh

```diff
@@ -62,6 +64,26 @@ RELAY_CONNECTIONSTRING="Endpoint=sb://${RELAY_NAME}.servicebus.windows.net/;Shar
 
 /anaconda/envs/py38_default/bin/pip3 install seaborn
 
+# Update rbase
+
+echo "Y"|sudo apt install --no-install-recommends r-base
+
+#Update kernel list
+
+echo "Y"| /anaconda/bin/jupyter kernelspec remove sparkkernel
+
+echo "Y"| /anaconda/bin/jupyter kernelspec remove sparkrkernel
+
+echo "Y"| /anaconda/bin/jupyter kernelspec remove pysparkkernel  
+
+echo "Y"| /anaconda/bin/jupyter kernelspec remove spark-3-python 
+
+echo "Y"| /anaconda/bin/jupyter kernelspec remove julia-1.6
```

Comment:
It makes me 😢 to remove Julia -- should we keep it?

(Makes sense to remove the Spark ones)

Reply:
> It makes me 😢 to remove Julia -- should we keep it?
> 
> (Makes sense to remove the Spark ones)

Julia is still on the VM but not in Kernel list. I updated the PR and keep Julia kernel.